lab1

Rugelgen opened the pull request.

Rugelgen committed:
Add files via upload

TimurIslamgulov commented on ЛР1/1.py:
```diff
@@ -0,0 +1,150 @@
+import doctest
+doctest.testmod()
+
+class fridge:
```
название классов пишем с большой буквы

TimurIslamgulov commented on ЛР1/1.py:
```diff
@@ -0,0 +1,150 @@
+# TODO Написать 3 класса с документацией и аннотацией типов
+from typing import Union
+import doctest
+doctest.testmod()
```
тест надо запускать внизу кода

TimurIslamgulov commented on ЛР1/1.py:
```diff
@@ -0,0 +1,150 @@
+        пареметры холодильник:
+        :param status: информация о том, закрыт холодильник или открыт
+        :param content: информация о содержании холодильника
+        >>>Xolodilnik_1 = fridge(False,слон)  # инициализация экземпляра класса
```
тут по логике уже просится список для содержимого

Rugelgen replied:
Анекдот про холодильник это классика.

TimurIslamgulov commented on ЛР1/1.py:
```diff
@@ -0,0 +1,150 @@
+        self.color = None
+
+        if not isinstance(status, Union[int,float]):
+            raise TypeError("Статус должен быть числовым значением")
```
почему нет проверки на отрицательное число?

Rugelgen replied:
Начальное значение задаётся 0. Далее методами get_it_money и put_money значение соответственно уменьшается или увеличивается.
Здесь скорее нужно ввести проверку в в методах, чтобы туда вводились только положительные числа, чтобы они работали корректно.

Rugelgen committed:
Update 1.py

исправлены предыдущие ошибки 
1) Все названия классов написаны с большой буквы
2)doctest.testmod() запущен внизу кода (а не вначале)
3) При инициализации объекта класса Wallet добавлена проверка на отрицательное число (дабы сумма в кошельке не могла быть менее 0)
